src/Desktop.Analyzers/Core/DesktopAnalyzersResources.resx

```diff
@@ -368,7 +368,7 @@
     <comment>CA3075</comment>
   </data>
   <data name="DoNotUseDtdProcessingOverloadsMessage" xml:space="preserve">
-    <value>{0} uses unsafe overload of '{1}' method</value>
+    <value>Uses unsafe overload of '{0}' method</value>
```

Comment:
We had an Email and I have you confirmed that we need to remove the first parameter messages as we using IOperation. It is no use since there is accurate location info in the issue.